### DISCUSSION TITLE
Grounded Opertation都有哪些类型？

### DISCUSSION THREAD

**Discussion by truebit** [THUDM/CogVLM#287] [Q&A]
1. 目前尝试在移动app上实验CogAgent，发现最常见的是`Grounded Operation:tap [[154,244]]`，还有`press home`, `press back`, `swipe up`等。能列举它可能出现的所有动作么？
2. 另外发现`Grounded Operation:press home [INST]`这种`[INST]`标识是什么含义呢？

**Comment by zRzRzRzRzRzRzR** [comment]
[INST] 是特殊字段啊，不应该被屏蔽掉吗

**Comment by truebit** [comment]
模型在history比较长的情况下，可能会输出这个字段

Br,

Sean Wang
Blog: fclef.wordpress.com <http://fclef.wordpress.com/about>


zR ***@***.***>于2024年1月9日 周二10:57写道：
